utils/email-helper.js

```diff
@@ -0,0 +1,60 @@
+const config = require('config');
+const sgMail = require('@sendgrid/mail');
+
+const recipient = config.get('recipient.to');
+const sender = config.get('sender.from');
+
+const sendEmail = async (message) => {
+    const sgapiKey = config.get('api_key.SENDGRID');
+    sgMail.setApiKey(sgapiKey);
```

Comment:
I think this `sgMail.setApiKey(sgapiKey)` doesn't need to be done everytime when `sendEmail` is called. 

We can move it to outside?

Reply:
I put this inside, because that seems to cause an error [here](https://github.com/nishi-ai/softball-register-server/blob/9b4685b4f22f7c97dd54c3dbb5c71abb7f2fb7f8/controllers/registerController.js#L3) like "Error: Configuration property "api_key.SENDGRID" is not defined" 🧐

Reply:
Ok, it's because there is no Env-var to obtain in our development environment whereas Heroku environment has. 
You can use `export SENDGRID_API_KEY={{KEY_BLAH}}` on the terminal and run the development server again.

But you have to run it every time when you open a new terminal, which is annoying. 
So additionally, we can put a dummy value like the following in the `default.json`. 
```
    "api_key" : {
        "SENDGRID": "xxx" (any random string)
    }
```
So when `node-config` can't find env-var to connect that are in `config/custom-environment-variables.json`, it would have a default value so the app won't crash. 

NOTE: Notice it's a dummy string, it can't send an email for testing. Then you can use `export SENDGRID_API_KEY=` with the real key together when you need to test time to time.